modules/aggregations/src/yamlRestTest/resources/rest-api-spec/test/aggregations/matrix_stats_multi_value_field.yml

```diff
@@ -132,10 +134,12 @@ setup:
   - match: {hits.total: 15}
   - match: {aggregations.mfs.doc_count: 14}
   - match: {aggregations.mfs.fields.0.count: 14}
-  - match: {aggregations.mfs.fields.0.correlation.val1: 0.06838646533369998}
+  - close_to: {aggregations.mfs.fields.0.correlation.val1: { value: 0.06838646533369998, error: 0.1 } }
```

Comment:
The `close_to` change is only needed to make the tests work for mixed cluster qa and runtime fields qa modules.

Reply:
This is because of a bug... the `mode` field isn't serialised in the builder. And defaults to `avg` and the test sets this to max. In the mentioned qa modules we do run with multiple nodes and so the `mode` was lost.